src/main/webapp/assets-lti/js/utils.js

```diff
@@ -55,8 +55,13 @@ var filterOutSections = function(sectionData, mPathArray){
   return sectionData;
 };
 
-var parseResultLookUpCanvasUser = function(userArray, uniqname){
-  var thisUser = _.findWhere(userArray, {sis_login_id: uniqname});
+var parseResultLookUpCanvasUser = function(userArray, uniqname, userIsFriend){
+  if(!userIsFriend){
+    var thisUser = _.findWhere(userArray, {sis_login_id: uniqname});
```

Comment:
Not sure that this belongs here but someone reading the code should know how to find out why it is sis_login_id sometimes and sis_user_id other times.